src/restconfig/src/main/java/org/geoserver/rest/catalog/SequentialExecutionController.java

```diff
@@ -0,0 +1,10 @@
+/* (c) 2022 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.rest.catalog;
+
+/* A marker interface.
```

Comment:
Still not a javadoc comment... sorry! This one is a common multiline comment, a javadoc one starts with ``/**`` instead. See also https://www.tutorialspoint.com/java/java_documentation.htm

Reply:
Yes, you are right! I will fix it right away.

Reply:
@aaime Javadoc created